smartcontract/service/native/governance/utils.go

```diff
@@ -232,9 +235,53 @@ func InitConfigBytes() []byte {
 	return bf.Bytes()
 }
 
-func splitCurve(pos uint64, avg uint64) uint64 {
-	xi := PRECISE * YITA * 2 * pos / (avg * 10)
+func GetOngBalance(native *native.NativeService, address common.Address) (uint64, error) {
+	buf := bytes.NewBuffer(nil)
+	err := serialization.WriteVarBytes(buf, address[:])
+	if err != nil {
+		return 0, errors.NewDetailErr(err, errors.ErrNoCode, "getOngBalance, serialization.WriteVarBytes error!")
+	}
+
+	value, err := native.ContextRef.AppCall(genesis.OngContractAddress, "balanceOf", []byte{}, buf.Bytes())
+	if err != nil {
+		return 0, errors.NewDetailErr(err, errors.ErrNoCode, "getOngBalance, appCall error!")
+	}
+	balance := new(big.Int).SetBytes(value.([]byte)).Uint64()
+	return balance, nil
+}
+
+func splitCurve(pos uint64, avg uint64, yita uint32) uint64 {
+	xi := PRECISE * uint64(yita) * 2 * pos / (avg * 10)
 	index := xi / (PRECISE / 10)
 	s := ((Yi[index+1]-Yi[index])*xi + Yi[index]*Xi[index+1] - Yi[index+1]*Xi[index]) / (Xi[index+1] - Xi[index])
 	return s
 }
+
+func GetUint32Bytes(view uint32) []byte {
+	var p [4]byte
+	binary.LittleEndian.PutUint32(p[:], view)
```

Comment:
please use serialization interface.  

same comments for the following GetBytesUint32.